README.md

```diff
@@ -123,6 +123,7 @@ In these labs you will use the Configurable App Templates, the ArcGIS Web App Bu
  * [Search with a Query Task](./develop/jsapi/search_with_query_task/lab.md)
  * [Client-side Buffering](./develop/jsapi/buffer_with_geometry_engine/lab.md)
  * [Client-side Buffering with UI](./develop/jsapi/buffer_with_geometry_engine_and_slider/lab.md)
+ * [Working with a Proxy](./develop/jsapi/work_with_proxy/lab.md)
```

Comment:
I did twirl around the title _Working with ArcGIS Online Proxy_ to try and differentiate from a self-hosted [resource-proxy](https://github.com/Esri/resource-proxy). I think the word "proxied" may end up being confusing since we don't use it anywhere else in our docs.